Emby.Server.Implementations/UserViews/FolderImageProvider.cs

```diff
@@ -1,3 +1,5 @@
+#pragma warning disable SA1649
```

Comment:
Why not rename this file instead?

Reply:
I don't think this was resolved, afaik it should be safe to rename this file to match the class